backend/app/views/spree/admin/orders/_risk_analysis.html.erb

```diff
@@ -1,50 +1,26 @@
-<fieldset class="no-border-bottom" id="risk_analysis">
-  <legend><%= "#{t('spree.risk_analysis')}: #{t('spree.not') unless @order.is_risky?} #{t('spree.risky')}" %></legend>
+<details id="risk_analysis">
+  <summary><%= t('spree.risk_analysis') %>: <%= t('spree.risky') %></summary>
+
   <table>
     <thead>
+      <th><%= t('spree.payment')%></th>
       <th><%= t('spree.risk')%></th>
       <th><%= t('spree.status')%></th>
     </thead>
     <tbody id="risk-analysis" data-hook="order_details_adjustments"  class="with-border">
-      <tr class="">
-        <td><strong>
-          <%= t('spree.failed_payment_attempts') %>:
-        </strong></td>
-        <td>
-          <span class="pill pill-<%= @order.payments.failed.count > 0 ? 'warning' : 'complete' %>">
-            <%= t(
-              'spree.payments_failed_count',
-              count: @order.payments.failed.count
-            ) %>
-          </span>
-        </td>
-      </tr>
-
-      <tr>
-        <td><strong><%= t('spree.avs_response') %>:</strong></td>
-        <td>
-          <span class="pill pill-<%= latest_payment.is_avs_risky? ? 'warning' : 'complete' %>">
-            <% if latest_payment.is_avs_risky? %>
-              <%= avs_response_code[latest_payment.avs_response] %>
-            <% else %>
-              <%= t('spree.success') %>
-            <% end %>
-          </span>
-        </td>
-      </tr>
-
-      <tr>
-        <td><strong><%= t('spree.cvv_response') %>:</strong></td>
-        <td>
-          <span class="pill pill-<%= latest_payment.is_cvv_risky? ? 'warning' : 'complete' %>">
-            <% if latest_payment.is_cvv_risky? %>
-              <%= cvv_response_code[latest_payment.cvv_response_code] %>
-            <% else %>
-              <%= t('spree.success') %>
-            <% end %>
-          </span>
-        </td>
-      </tr>
+      <% @order.payments.risky.each do |payment| %>
```

Comment:
Nit: Have you considered moving the `@order` instance variable out of the partial? From https://andycroll.com/ruby/dont-use-instance-variables-in-partials/: 

![image](https://user-images.githubusercontent.com/61476/216266587-351fdff0-9a7c-47c9-815e-01418f9d898b.png)

Reply:
I'm deeply aware of this problem, [in fact have been for 10 years already](https://www.slideshare.net/elia.schito/rails-oo-views). In this case I would say it's a rather benign situation and the whole risk support is not very useful currently, but I'll pass the order from outside 👍 

BTW Rails 7.1 will allow listing accepted locals for partials, which will alleviate the biggest issue with using locals 🙌